run/send-push-message

```diff
@@ -23,6 +23,14 @@ function send_gotify () {
     "$GOTIFY_DOMAIN/message?token=$GOTIFY_APP_TOKEN"
 }
 
+function send_discord() {
+  log "Sending Discord message."
+
+  curl -H "Content-Type: application/json" -d \
+    '{"username": "TeslaUSB", "content": "'"$message"'"}' \
+    $DISCORD_WEBHOOK_URL
```

Comment:
shellcheck wants double quotes around the url var

Reply:
@marcone added